server/.env

```diff
@@ -1 +1,4 @@
 TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Jr"
+MONGO_URI=mongodb+srv://KimImpossible:Kimmy22@flightsdb.dsvmk.mongodb.net/FlightsDB?retryWrites=true&w=majority
+AWS_ACCESS_KEY="AKIASOA4UQQKYCYFAAUP"
```

Comment:
This is what Chris was mentioning about the keys being exposed: never include your environment file in your commits.